lib/platforms.nix

```diff
@@ -8,8 +8,9 @@ rec {
   openbsd = ["i686-openbsd" "x86_64-openbsd"];
   netbsd = ["i686-netbsd" "x86_64-netbsd"];
   cygwin = ["i686-cygwin" "x86_64-cygwin"];
-  unix = linux ++ darwin ++ freebsd ++ openbsd;
-  all = linux ++ darwin ++ cygwin ++ freebsd ++ openbsd;
+  illumos = ["x86_64-solaris"];
```

Comment:
Sure, but on nix side I assume we want to handle all these variants the same way, don't we? If so, it would seem better to use more consistent naming to indicate that. (Otherwise people might e.g. think that we differentiate between them.)